README.md

```diff
@@ -1,5 +1,10 @@
 # Postgres Test
 
-This repo contains one branch per language for the cockroach acceptance tests.
-These are auto-built on push and tagged on the docker hub at
-https://hub.docker.com/r/cockroachdb/postgres-test/.
+This is the Docker image used to run the CockroachDB
+[acceptance tests](https://github.com/cockroachdb/cockroach/tree/master/acceptance),
+which include verification that simple clients in various languages can connect
+to and interact with CockroachDB using the Postgres wire protocol -- thus it
+includes the tools required to run php, node, java, etc.
+
+[Dockerhub](https://hub.docker.com/r/cockroachdb/postgres-test/) automatically builds new images from git tags, using the git tag as
```

Comment:
s/Dockerhub/Docker Hub/

also, did you mean to wrap this line as you do above? wrapping should be done as appropriate for source, rather than the rendered result (which ignores single line breaks anyway).